File: backend/src/main/java/dev/anthonybruno/gymbuddy/auth/SessionUtils.java
package dev.anthonybruno.gymbuddy.auth;

import dev.anthonybruno.gymbuddy.user.model.UserDetails;
import io.javalin.http.Context;

public class SessionUtils {

    public static String AUTH_ATTR_KEY = "auth";

    public static void setSession(Context context, UserDetails userDetails) {
        context.sessionAttribute(AUTH_ATTR_KEY, userDetails);
    }

    public static UserDetails getSession(Context context) {
        return context.sessionAttribute(AUTH_ATTR_KEY);
    }
}

File: backend/src/main/kotlin/dev/anthonybruno/gymbuddy/auth/SessionUtils.kt
package dev.anthonybruno.gymbuddy.auth

import dev.anthonybruno.gymbuddy.auth.SessionUtils.AUTH_ATTR_KEY
import dev.anthonybruno.gymbuddy.user.UserDetails
import io.javalin.http.Context

object SessionUtils {

    const val AUTH_ATTR_KEY = "auth"

    fun setSession(context: Context, userDetails: UserDetails) {
        context.sessionAttribute(AUTH_ATTR_KEY, userDetails)
    }

}
